HISTORY.rst

```diff
@@ -27,6 +27,7 @@
 
 **Bug fixes**
 
+- 1093_: [SunOS] memory_maps() shows wrong 64 bit addresses
```

Comment:
Was there also a mismatch with the other numbers?  You change the type of all of them.